lib/elixir/lib/process.ex

```diff
@@ -26,19 +26,15 @@ defmodule Process do
   Inlined by the compiler.
   """
   @spec alive?(pid) :: boolean
-  def alive?(pid) do
-    :erlang.is_process_alive(pid)
-  end
+  defdelegate alive?(pid), to: :erlang, as: :is_process_alive
 
   @doc """
   Returns all key-value pairs in the process dictionary.
 
   Inlined by the compiler.
   """
-  @spec get :: [{term, term}]
-  def get do
-    :erlang.get()
-  end
+  @spec get() :: [{term, term}]
+  defdelegate get(), to: :erlang
```

Comment:
Well, I guess this is one place you actually don't need to use `()`. Also, the other places ([like this](https://github.com/elixir-lang/elixir/blob/3821dd827b63bf2d4846cfdc4fbfb219390b78e2/lib/mix/lib/mix/shell/quiet.ex#L15)) were `defdelegate` is used to a 0 arity function, the `()` is omitted. IMO, it would be good to maintain the consistency over the many ocurrencies of it, but maybe is my OCD talking too loud. 😬